### PR TITLE
Mem 118 convert feed ellipsis icon

### PR DESCRIPTION
**Pull request by jacksonopp**
This PR replaces the ellipses icon on the feed view with dynamic icons that represent which listing type you are viewing.

## Subscribed
Simulator Screenshot - iPhone SE (3rd generation) - 2023-06-30 at 12 24 32

## Local
Simulator Screenshot - iPhone SE (3rd generation) - 2023-06-30 at 12 24 18

## All
Simulator Screenshot - iPhone SE (3rd generation) - 2023-06-30 at 12 23 59

### CODE REVIEW

**Comment by jacksonopp** [comment]
This addresses #255

File: components/ui/Feed/FeedView.tsx [hunk 1]
```diff
@@ -48,12 +51,18 @@ const SortIconType = {
...
+  Local: <IconMapPin />,
+  Subscribed: <IconBookmark />
+};
+
 function FeedView({ feed, community = false, header }: FeedViewProps) {
   // State Props
   // TODO Handle this
   // eslint-disable-next-line @typescript-eslint/no-unused-vars
   const [endReached, setEndReached] = useState(false);
-  const [, setSortIcon] = useState(SortIconType[feed.sort]);
```
Comment (jacksonopp) [inline components/ui/Feed/FeedView.tsx:56]: I believe the `useEffect` on line 83 is what actually causes the re-render, so I think deleting this state setter is no longer needed?

Comment (gkasdorf) [reply components/ui/Feed/FeedView.tsx:56]: Yea I think that was an old remnant. Good to get rid of it. (Lots of stuff to clean up as you can probably tell, that's the next item on the agenda really before feature implementation)

**Comment by gkasdorf** [comment]
Looks good. I'll take a look at this later whenever I'm home. Thanks!

**Review by gkasdorf — APPROVED** [review]

**Pull request merged**